### DISCUSSION TITLE
Quiet flag for tailwindcss cli

### DISCUSSION THREAD

**Discussion by clintjedwards** [tailwindlabs/tailwindcss#13439] [Ideas]
Hey all! 

I'm loving tailwind and have integrated it into most of my personal projects via a simple make command that runs: `npx tailwindcss -i <input> -o <output> --watch` while I code so I can have live rebuilds. My problem lies in that the above command will output to stderr the messages "Rebuilding..." and "Done in [x]ms". This is nice, but the make command runs the above command as part of how the application runs also. So my application logs will output in the same terminal session as the tailwindcss cli output.

This gets to be a bit messy as the logs end up being broken up by tailwindcli's rebuilding messages, which honestly I typically don't care about unless there is an error(although I can see a good reason for being able to quiet errors as well).

My questions are:

1. Is there a way to suppress the output for Tailwind CLI? or at least get the common output to be sent to stdout instead of stderr (This is because I usually want to hear about errors but would like to avoid the constant "Rebuilding"...). 
2. Before I start writing any code would a flag like `--quiet` be a reasonable addition? I understand new features aren't considered very often from the outside, but this seems like a (reasonably) quick win that I would be willing to post a PR for.
3. I took a look at the code and I found the output location for "Done in [x]ms" but I couldn't find where the "Rebuilding..." string is coming from. I'm pretty junior at javascript/typescript so I just may be misunderstanding where to find it (possible it's in the watcher lib?).

Redirecting the common output seems like the quickest win here, although technically it is a breaking change.

Thanks for your time!

**Comment by grempe** [comment]
Related:

https://github.com/phoenixframework/tailwind/issues/128
